### PR TITLE
Remove old ICDS dashboard feature flag

### PR DESCRIPTION
**Pull request by emord**
tableau hasn't existed for quite a while.

### CODE REVIEW

File: custom/icds_reports/views.py [hunk 1]
```diff
@@ -121,7 +121,7 @@
 
 
 @location_safe
-@method_decorator([toggles.ICDS_REPORTS.required_decorator(), login_and_domain_required], name='dispatch')
+@method_decorator([login_and_domain_required], name='dispatch')
```
Comment (emord) [inline custom/icds_reports/views.py]: It's likely no one uses this, but will go through nginx logs before I remove the redirect

**Review by orangejenny — APPROVED** [review]
❤️

**Comment by orangejenny** [comment]
Test failures look real, though: `ImportError: cannot import name IcdsBaseReport`

**Comment by emord** [comment]
:white_check_mark: (thanks for removing the JS)

**Review by orangejenny — APPROVED** [review]

**Pull request merged**